### ISSUE TITLE
Puzzlebot Position in Obstacle World

### ISSUE THREAD

**Issue by A01379375** [ManchesterRoboticsLtd/TE3003B_Integration_of_Robotics_and_Intelligent_Systems#2]
Greetings,

My team (Los Súbditos de Kauil) and I found a small issue with how the Puzzlebot was spawned in the obstacle world, compared to the track world. In the track world, **the floor** is spawned 10 cm below the Puzzlebot (exactly 1 diameter of its wheels), however, in the obstacle world it is spawned in position (0,0,0). Since in both cases the robot is spawned also in (0,0,0), this made it so that in the obstacle world it was spawned **inside the floor**. It only lasted for a few milliseconds at most, but it was enough to destroy the publishing of wl and wr (adding a constant error of 0.002 rad/s). To solve it, we established a new launch file [puzzlebot_obstacle_world.zip](https://github.com/ManchesterRoboticsLtd/TE3003B_Integration_of_Robotics_and_Intelligent_Systems/files/11409534/puzzlebot_obstacle_world.zip) in which the Puzzlebot is always spawned 10 cm above (0,0,0.1), ensuring it will never be inside the ground in either world. This completely solved the issue.

TL;DR: Fixed Puzzlebot spawning inside the ground in obstacle.world
